versions/3.0.md

```diff
@@ -1184,6 +1185,23 @@ Locating a linked resource may be performed by either a `href` or `operationId`.
 In the case of an `operationId`, it must be unique and resolved in the scope of the OAS document.
 Because of the potential for name clashes, consider the `href` syntax as the preferred method for specifications with external references.
 
+When the `url` attribute is provided, the target host may be overridden by the value provided.  For example:
+
+```
+components:
+  links:
+    UserRepositories:
+      operationId: getRepositoriesByOwner
+      url: $responseHeaders.link
```

Comment:
Also, the examples in the table below have the variables in braces (i.e. `{$responseHeader.link}`, here they are raw. I think we should always use the braces, or at least define it clearly when they can be omitted.